PrismLibrary_Avalonia.slnf

```diff
@@ -0,0 +1,12 @@
+{
+  "solution": {
+    "path": "PrismLibrary.sln",
+    "projects": [
+      "src\\Containers\\Prism.DryIoc.Shared\\Prism.DryIoc.Shared.shproj",
+      "src\\Avalonia\\Prism.Avalonia\\Prism.Avalonia.csproj",
+      "src\\Avalonia\\Prism.DryIoc.Avalonia\\Prism.DryIoc.Avalonia.csproj",
+      "tests\\Avalonia\\Prism.Avalonia.Tests\\Prism.Avalonia.Tests.csproj",
+      "tests\\Avalonia\\Prism.DryIoc.Avalonia.Tests\\Prism.DryIoc.Avalonia.Tests.csproj"
```

Comment:
this would need to be updated

Reply:
Thank you Dan for the quick response on this one. I was curious about the best strategy to move forward with regarding that. In the first round, I was planning on getting a solid build running and then minimizing if that's okay.

An example of super-minor differences is WPF's `DependencyObject` vs. Avalonia's `AvaloniaObject`.

I'll take a look into the Uno.WinUI implementation, thank you! Currently, I'm working on the importation of the tests